utils/scripts/ocm/ocm.py

```diff
@@ -412,6 +411,120 @@ def uninstall_rhods(self):
         """Uninstalls RHODS addon"""
         self.uninstall_addon(addon_name="managed-odh")
 
+    def install_addon_v2(self, addon_name="managed-odh",
+                         template_filename="install_addon.jinja",
+                         output_filename="install_operator.json",
+                         add_replace_vars=None):
+        """Installs addon"""
+        replace_vars = {
+                       "CLUSTER_ID": self.cluster_name,
+                       "ADDON_NAME": addon_name
+                       }
+        if add_replace_vars:
+            replace_vars.update(add_replace_vars)
+            print(replace_vars)
+        template_file = template_filename
+        output_file = output_filename
+        self._render_template(template_file, output_file, replace_vars)
+        cluster_id = self.get_osd_cluster_id()
+        cmd = ("ocm post /api/clusters_mgmt/v1/clusters/{}/addons "
+               "--body={}".format(cluster_id, output_file))
+        log.info("CMD: {}".format(cmd))
+        ret = execute_command(cmd)
+        print(ret)
+        failure_flag = False
+        if ret is None:
+            log.info("Failed to install {} addon on cluster "
+                  "{}".format(addon_name, self.cluster_name))
+            failure_flag = True
+            return failure_flag
+            # sys.exit(1)
+        return failure_flag
+
+    def install_rhoam_addon(self):
+        def check_secret_existence(secret_name):
+            cmd = "oc get secret {} -n redhat-rhoam-operator".format(secret_name)
```

Comment:
Make this as a common method and get operator name as an argument, so that it will be generic